static/app/components/tag.tsx

```diff
@@ -95,6 +96,15 @@ function Tag({
     onDismiss?.();
   }
 
+  const trackClickEvent = () => {
+    trackAnalyticsEvent({
```

Comment:
trackAnalyticsEvent is deprecated we should try and use the trackAdvancedAnalyticsEvent if possible.